docs/program/sdks/_index.md

```diff
@@ -204,6 +205,43 @@ async function main() {
 main();
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+```cpp {class="line-numbers linkable-line-numbers"}
+# include <string>
+# include <vector>
+
+# include <boost/optional.hpp>
+
+# include <viam/api/common/v1/common.pb.h>
+# include <viam/api/robot/v1/robot.grpc.pb.h>
+
+# include <viam/sdk/robot/client.hpp>
+# include <viam/sdk/components/camera/client.hpp>
+
+using namespace viam::sdk;
+
+int main() {
+  std::string host("guardian-main.vw3iu72d8n.viam.cloud");
```

Comment:
I don't think we should include an actual uri/credentials here, should we?

Reply:
```suggestion
  std::string host("ADDRESS FROM THE VIAM APP");
```